04.ls/ls_command.rb

```diff
@@ -5,47 +5,50 @@
 COLUMN_COUNT = 3
 
 def main
-  files = if a_option?
-            fetch_files(option: 'a')
-          else
-            fetch_files
-          end
-  puts format_to_show(files)
+  options = load_options
+  sorted_files = fetch_files.sort
+
+  filtered_files = if options.key?(:a)
+                     sorted_files
+                   else
+                     sorted_files.reject { |file| file.start_with?('.') }
+                   end
+
+  reversed_files = if options.key?(:r)
```

Comment:
```ruby
files = fetch_files.sort
```

として、

```suggestion
  sorted_files = options.key?(:r) ? filtered_files.reverse : filtered_files
```

とした方が意図が伝わりやすい気がしました。
（最初は「ただのファイル（`files`）」があって、それをrオプションの有無によって適切に並び替える（sort）するイメージです）